src/coreComponents/constitutive/permeability/CarmanKozenyPermeability.hpp

```diff
@@ -143,7 +149,7 @@ void CarmanKozenyPermeabilityUpdate::compute( real64 const & porosity,
                                               arraySlice1d< real64 > const & permeability,
                                               arraySlice1d< real64 > const & dPerm_dPorosity ) const
 {
-  real64 const constant = pow( m_sphericity*m_particleDiameter, 2 ) / 180;
+  real64 const constant = pow( m_sphericity*m_particleDiameter, 2 ) / 150;
```

Comment:
Is this a bug fix?

Reply:
Based on the experimental results and assumption of ideally straight flow channels, Carman (1937) initially proposed Kc = 180. By taking the fluid tortuosity into consideration, Carman (1956) further modified this linear relationship with Kc = 150, which was referenced by the [online document](https://geosx-geosx.readthedocs-hosted.com/en/latest/coreComponents/constitutive/docs/KozenyCarmanPermeability.html?_sm_au_=iMVDJfSRJqbfqtLRQ0WpHK6H8sjL6).